tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -33,7 +33,7 @@ def test_lambda_w_dynamodb_event_filter(deploy_cfn_template, aws_client):
             "FunctionName": function_name,
             "TableName": table_name,
             "Filter": '{"eventName": ["MODIFY"]}',
-        },
+        }
```

Comment:
probably a typo causing preflight linting to fail.

Hint: double-check the pre-commit hooks and/or IDE linter settings. They usually catch these, but I've also had some escaping into CI (especially around linter updates) :)

Reply:
Fixed